l5kit/l5kit/data/map.py

```diff
@@ -54,37 +54,32 @@ def proto_to_semantic_map(map_fragment: "MapFragment") -> dict:
             lane = element.element.lane
 
             # get left-right coordinates and element id
-            x_left, y_left, z_left = unpack_boundary(lane.left_boundary, lane.geo_frame)
-            x_right, y_right, z_right = unpack_boundary(lane.right_boundary, lane.geo_frame)
-            lanes.append(
-                {
-                    "xyz_left": (x_left, y_left, z_left),
-                    "xyz_right": (x_right, y_right, z_right),
-                    "id": element.id.id.decode("utf-8"),
-                }
-            )
+            xyz_left = unpack_boundary(lane.left_boundary, lane.geo_frame, ecef_to_pose)
+            xyz_right = unpack_boundary(lane.right_boundary, lane.geo_frame, ecef_to_pose)
+            lanes.append({"xyz_left": xyz_left, "xyz_right": xyz_right, "id": element.id.id.decode("utf-8")})
+
             # store bounds for fast rasterisation look-up
-            x_min = min(np.min(x_left), np.min(x_right))
-            y_min = min(np.min(y_left), np.min(y_right))
-            x_max = max(np.max(x_left), np.max(x_right))
-            y_max = max(np.max(y_left), np.max(y_right))
+            x_min = min(np.min(xyz_left[:, 0]), np.min(xyz_right[:, 0]))
```

Comment:
What's benefits of packing (x,y,z) together? I prefer the old way which separete them, it is easier to understand